puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/packaging_build_deb_plugin.yaml

```diff
@@ -12,7 +12,7 @@
           type: user-defined
           name: os
           values:
-          - xenial
+          - buster
```

Comment:
Do we want to stay on Ubuntu based machines for plugin building?

Reply:
Yes, because bionic is a tad older than bionic and we need to stay compatible.